webpack-plugin/package.json

```diff
@@ -43,6 +43,7 @@
 		"webpack-cli": "^4.9.1"
 	},
 	"dependencies": {
-		"loader-utils": "^2.0.2"
+		"loader-utils": "^2.0.4",
+		"json5": "2.2.3"
```

Comment:
As I pointed out, I am in doubt that this works, Did you build this package and checked the artifact produced?